### PR TITLE
fix(cesium): missing objects added

### LINKED ISSUE #42880: @types/cesium Missing types for MapboxImageryProvider and MapboxStyleImageryProvider
- [ ] I tried using the `@types/xxxx` package and had problems.
- [ ] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [x] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Please ask any appropriate questions there).
- [x] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @Zuzon @hnipps @szechyjs @golyalpha @KeyboardSounds @soyarsauce @jrafidi @morgansierrasnyder @crocsx 

I could not find the types for [MapboxImageryProvider](https://cesium.com/docs/cesiumjs-ref-doc/MapboxImageryProvider.html) and [MapboxStyleImageryProvider](https://cesium.com/docs/cesiumjs-ref-doc/MapboxStyleImageryProvider.html). Are these on the roadmap for future releases?

Thanks.

### PR DESCRIPTION
**Pull request by peterblazejewicz**
- `MapboxStyleImageryProvider`
https://cesium.com/docs/cesiumjs-ref-doc/MapboxStyleImageryProvider.html
- `MapboxImageryProvider`
https://cesium.com/docs/cesiumjs-ref-doc/MapboxImageryProvider.html

Fixes: #42880

Thansk!

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://cesium.com/docs/cesiumjs-ref-doc/MapboxStyleImageryProvider.html 
https://cesium.com/docs/cesiumjs-ref-doc/MapboxImageryProvider.html
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/cesium/index.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for cesium 1.66
+// Type definitions for cesium 1.67
```
Comment (jrafidi) [inline types/cesium/index.d.ts]: It doesn't look like this PR covers all of the additions/removals in 1.67 [1], so I don't think we should bump this yet.

1: https://github.com/CesiumGS/cesium/blob/master/CHANGES.md#1670---2020-03-02

**Review by jrafidi — COMMENTED** [review]
Mapbox types look correct, not sure about the version bump.

File: types/cesium/index.d.ts (hunk 1 above)
Comment (peterblazejewicz) [reply types/cesium/index.d.ts]: Reverted that change, thanks for the review!

**Review by jrafidi — APPROVED** [review]

**Pull request merged**